client/src/App.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import TaskBox from './TaskBox'
 
 function App() {
   return (
+    /*
```

Comment:
This should be a multi-line commit feature.

Since we're currently in the component development phase, we want to make sure that we don't render anything as of yet. As you're developing, feel free to include it in here, but when submitting a PR, try to remove it so there aren't any Merge Conflicts 👍🏽 

Testing out the Multi-line Commit Feature